tests/performance/apdex-score/README.md

```diff
@@ -2,13 +2,46 @@
 
 ## Setup
 
+#### Prerequisites
+
+Before continuing with the setup steps below, ensure you have a cht instance deployed and running either locally or globally - check out the [documentation](https://docs.communityhealthtoolkit.org/contribute/code/core/dev-environment/) on how to do this.
```

Comment:
This one will not work because if you use the development setup you will have an instance that doesn't contain the certificates, so you will not be able to access it through the phone. I am sorry, this was my suggestion and I completely forgot about it. 
Please change the link to point to the [docker-helper](https://docs.communityhealthtoolkit.org/hosting/4.x/app-developer/) instructions, that instance will be accessible from the phone using the correct ip address and not the local one (`127.0.0.1`)

Reply:
sure thanks